arches_lingo/src/arches_lingo/components/schemes/SchemeBox.vue

```diff
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+import { routeNames } from "@/arches_lingo/routes.ts";
+
+// TODO: Shift into types.ts
+interface Scheme {
+    resourceinstanceid: string;
+    descriptors: {
+        [key: string]: {
+            name: string;
+            description: string;
+        };
+    };
+}
+
+const { scheme } = defineProps<{ scheme: Scheme }>();
+const schemeURL = {
+    name: routeNames.scheme,
+    params: { id: scheme.resourceinstanceid },
+};
+</script>
+
+<template>
+    <div>
+        <RouterLink :to="schemeURL">
```

Comment:
RouterLink should enclose everything, yeah? Like no matter where they click the box it navs them there